.gitignore

```diff
@@ -55,4 +55,5 @@ hs_err_pid*
 application-default.*
 
 /src/main/resources/application.properties
-/src/main/resources/cert/teamssender.key
\ No newline at end of file
+/src/main/resources/cert/teamssender.key
+/src/main/resources/application.yml
```

Comment:
application.yml does not belong in .gitignore

(if you need to put secrets in a file for local testing: use .env)